api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -260,7 +260,8 @@
     public static final String OUTOFBANDMANAGEMENT_POWERSTATE = "outofbandmanagementpowerstate";
     public static final String OUTOFBANDMANAGEMENT_ENABLED = "outofbandmanagementenabled";
     public static final String OUTPUT = "output";
-    public static final String OVF_PROPERTIES = "ovfproperties";
+    public static final String PROPERTIES = "properties";
+    public static final String ACS_PROPERTY = "ACS-property";
```

Comment:
Is ACS_PROPERTY used by any API response?

Reply:
No, thanks. Moved its usage to the property prefix defined on ImageStore class